### PR TITLE
123.1 Chaos Game

### PR DESCRIPTION
**Pull request by kfahn22**
adding 123.1-chaos-game

### CODE REVIEW

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    ],
+    "groupLinks": [
+        {
+            "title": "References",
+            "links": [
+                {
+                    "icon": "📚",
+                    "title": "Algorithm Archive",
+                    "url": "https://www.algorithm-archive.org/",
+                    "description": "Arcane Algorithm Archive"
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:36]:
```suggestion
                    "description": "The Arcane Algorithm Archive is a collaborative effort to create a guide for all important algorithms in all languages. "
```

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                },
+                {
+                    "icon": "🔗",
+                    "title": "Chaos Game on Wikipedia",
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:40]:
```suggestion
                    "title": "Chaos Game",
```

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    "url": "https://en.wikipedia.org/wiki/Chaos_game",
+                    "description": "Discussion of the chao game."
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:42]:
```suggestion
                    "description": "Wikipedia page with background and references on the Chaos Game."
```

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                },
+                {
+                    "icon": "💿",
+                    "title": "Refactor by Espen Sande Larsen",
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:46]:
```suggestion
                    "title": "Refactor Song",
```

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                    "url": "https://soundcloud.com/espen-sande-larsen-365984601/refactor",
+                    "description": "Refactor song."
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:48]:
```suggestion
                    "description": "'I will refactor this later, a song by Espen Sande Larsen"
```

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+                }
+            ]
+        },
+        {
+            "title": "Videos",
+            "links": [
+                {
+                    "icon": "🚂",
+                    "title": "Chaos Game Part 2",
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:57]: This should go in related challenges? See #262

File: content/videos/challenges/123.1-chaos-game/index.json [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+                    "url": "/challenge/123.2-chaos-game",
+                    "description": "In the second part of the \u201cChaos Game\u201d coding challenge, I vary the number of seed points, probability, and percentage moved to generate other fractal patterns."
+                },
+                {
+                    "icon": "🎥",
+                    "title": "LeoisOS",
+                    "url": "https://www.youtube.com/LeiosOS",
+                    "description": "Intro to arcane algorithm archive."
```
Comment (shiffman) [inline content/videos/challenges/123.1-chaos-game/index.json:65]:
```suggestion
                    "description": "YouTube channel by the creator and maintainer of the Algorithm Archive."
```

**Review by shiffman — COMMENTED** [review]
Noting some edits and comments! Thank you @kfahn22!

File: content/videos/challenges/123.1-chaos-game/index.json (hunk 6 above)
Comment (kfahn22) [reply content/videos/challenges/123.1-chaos-game/index.json:57]: Yes, I wasn't sure how do deal with this.  I can change it back.

**Comment by kfahn22** [comment]
I changed the url for part 2 to youTube url.  I have part 2 already done and as soon as you merge I will undo this when adding part two if that is ok.

**Review by klinegareth — CHANGES_REQUESTED** [review]
Just noting that contribution 4 is missing (Customize the Chaos Game) as well as a thumbnail for contribution 8!

**Comment by kfahn22** [comment]
I just added an image for contribution8-- sorry I missed it.  Contribution3 gets a 404 and when I try to pull up contribution4 it doesn't seem to be working properly for me.  I started out renumbering the contributions when there was a broken or missing one but then decided that maybe it was unnecessary.

**Comment by kfahn22** [comment]
I have added contribution4 but the image is poor b/c the controls are not working for me (Mac-safari).  Maybe someone else can grab an image.  Please let me know if there is anything else I can do.

**Comment by klinegareth** [comment]
@kfahn22 I can add it!

**Review by klinegareth — APPROVED** [review]

**Comment by shiffman** [comment]
@klinegareth is this one ready to merge?

**Comment by klinegareth** [comment]
Not quite sure what the best way is for me to add these, but @kfahn22 here's screenshots for contribution 2 contribution 2 and contribution 4
contribution 4

**Comment by kfahn22** [comment]
How did you grab the image for contribution4?  Why browser did you use?  Should I consolidate part 1 and 2 of the challenge?

**Comment by klinegareth** [comment]
@kfahn22 I was on Firefox with the Dark Reader extension, and it wouldn't let me take a screenshot directly of the page so I just took one of my whole screen and cropped it! I think it would be good to add a link to the second part of the challenge to an `Other Parts of this Challenge` section in `groupLinks`. That seems like the best way to add multi-part challenges going forward!

**Pull request closed without merging**